backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -11,18 +16,161 @@ export class AdminResolver {
   async searchUsers(@Arg('searchText') searchText: string): Promise<UserAdmin[]> {
     const loginUserRepository = getCustomRepository(LoginUserRepository)
     const loginUsers = await loginUserRepository.findBySearchCriteria(searchText)
-    const users = loginUsers.map((loginUser) => {
-      const user = new UserAdmin()
-      user.firstName = loginUser.firstName
-      user.lastName = loginUser.lastName
-      user.email = loginUser.email
-      user.creation = [
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-      ]
-      return user
-    })
+    const users = await Promise.all(
+      loginUsers.map(async (loginUser) => {
+        const user = new UserAdmin()
+        user.firstName = loginUser.firstName
+        user.lastName = loginUser.lastName
+        user.email = loginUser.email
+        user.creation = await getUserCreations(loginUser.id)
+        return user
+      }),
+    )
     return users
   }
+
+  @Query(() => [Number])
```

Comment:
This is without any doubt a mutation

Reply:
Yeah, I will fix this I just made some copy and paste to create the methods, and the first one was a query...

Thanks for the catch!!!!!